selenium_tests/cross_listing/page_objects/cross_listing_index_page.py

```diff
@@ -0,0 +1,12 @@
+from selenium.webdriver.common.by import By
+
+from selenium_tests.cross_listing.page_objects.cross_listing_base_page_object\
+    import CrossListingBasePageObject
+
+
+class Locators(object):
+    HEADING_ELEMENT = (By.XPATH, '//h3[contains(.,"Cross Listing")]')
```

Comment:
This certainly works but I think getting by ID would be better. I think it's totally reasonable to go ahead and update the template being tested by adding id's. Not something that needs to happen on this, just food for thought.